### ISSUE TITLE
Readme package version & Accessibility

### ISSUE THREAD

**Issue by Splines** [marc-thieme/frame-it#7]
In the Readme, the quickstart says to import version `1.0.0` but with this version the `frames()` function does not exist. It should read `1.1.0`. It'd be also great to publish these versions as GitHub releases.

Furthermore, while I really appreciate the visuals in the Readme to showcase what the package does, this is particularly bad for accessibility. E.g. on the [typst site](https://typst.app/universe/package/frame-it/) I cannot copy anything from the Readme, nor search in it with the browser. Instead, I have to manually head to this GitHub repo, open your [Readme.typ](https://github.com/marc-thieme/frame-it/blob/main/README.typ) file and search for the corresponding code, which is a bit cumbersome and also hinders indexing the documentation itself in search engines.

**Comment by marc-thieme** [comment]
Hi, thanks for the issue and particularly for the feedback! I didn't know that it was not searchable embedded like that.

I fixed the issue with the version.

For the next release, I will also leverage the new `html` support and make the readme consist of the embedded html directly.
What's difficult there is currently that the native html can not handle my boxes. At least at the moment.

Therefore, I will have to write custom html in order to have my boxes exported correctly. Unfortunately, using `frame` does not work because GitHub doesn't render embedded svg.

As I'm constrained in time at the moment, I don't know when I will get to it.

**Comment by marc-thieme** [comment]
I reworked the entire README system in my commits from yesterday. I implemented html rendering.
Unfortunately, I was not aware of the fact that GitHub Markdown allows html but no css. For that reason, I am currently still generating all of my frames as svg and automatically embedding them into the markdown README.

However, I added an alert at the beginning of the README which redirects to the standalone html render of the README.
There, it is also possible to copy text finally.

Unfortunately, I messed up the `divide` line in the `boxy` style, which is now not aligned anymore.
I fixed it already in main but I want to wait and see if I need to do more fixes before I release the fixed version.
I hope that there are only very few people who will upgrade and need to use the divider in the next 2 weeks.

I am also aware that I forgot to up the version in the README again so I hope not many people will upgrade and see the regression anyway.
